cpu/cortexm_common/include/cpu.h

```diff
@@ -58,16 +58,79 @@ extern "C" {
  */
 #define PROVIDES_PM_SET_LOWEST
 
+/**
+ * @name   Pattern to write into the co-processor Access Control Register to
```

Comment:
Apparently `@name` opens a new group (at least that's how it looks like in the rendered doc). I also think this should be a `@brief` instead. If you intended to open a group, than please choose a shorter name and also close it.